tox.ini

```diff
@@ -1,31 +1,14 @@
 [tox]
-envlist = py38-django{22,30,31}
+envlist = py38-django{22,30,31,32}
 
 [doc8]
-; D001 = Line too long
 ignore = D001
 
 [pycodestyle]
 exclude = .git,.tox,migrations
 max-line-length = 120
 
 [pydocstyle]
-; D101 = Missing docstring in public class
-; D200 = One-line docstring should fit on one line with quotes
-; D203 = 1 blank line required before class docstring
```

Comment:
are these removed by running the `modernizer` script?

Reply:
Yes